lb_film_and_user_recommender.py

```diff
@@ -217,35 +216,43 @@ def get_film_recommendations(similar_users, my_profile, num_of_recs):
 
 ########################################################################################################################
 
-myprofile = 'tsar'
+valid_profile = False
+while not valid_profile:
+    myprofile = input("Please enter a Letterboxd username or press Enter to quit: ").strip()
+    if not myprofile:
+        print("Quitting...")
+        break
```

Comment:
Why not just exit the program so you don't have to nest all the rest in an if? 🤔